### DISCUSSION TITLE
What's the status of org-mode support?

### DISCUSSION THREAD

**Discussion by XaydBayeck** [srid/emanote#434] [Ideas]
I'm new guy for `emanote`. 

I try use `org-mode` as my base note format.

Well, i found `begin_center` is work, but I don't know how to use a **custom css** like the guide
```markdown
:::{.sticky-note}
   xxxx
:::
```

I wonder know how to do it in `org-mode`. Is it supported? 
And What's the status of org-mode support?

**Comment by srid** [comment]
I don't use org-mode myself. I'd say, if you are interested, feel free to open PRs, and I'd be happy to review and merge!

**Comment by srid** [comment]
It was added a while ago: https://emanote.srid.ca/guide/orgmode
